orval.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'orval';
+
+export default defineConfig({
```

Comment:
maybe we could think about a filter, for only those api's we really use here? 
https://orval.dev/reference/configuration/input#tags

Reply:
good one, I will check